commands/pull.go

```diff
@@ -16,7 +17,16 @@ import (
 
 // Handles the process of checking out a single file, and updating the git
 // index.
-func newSingleCheckout() *singleCheckout {
+func newSingleCheckout(gitEnv config.Environment, remote string) abstractCheckout {
+	manifest := getTransferManifestOperationRemote("download", remote)
+
+	//
+	clean, ok := gitEnv.Get("filter.lfs.clean")
+	if !ok || len(clean) == 0 {
+		return &noOpCheckout{manifest: manifest}
+	}
+	// */
```

Comment:
Should these empty comments be removed?

Reply:
Ahh yes, thanks. I was using that to ensure the tests were testing things correctly :)